Setup_Readme.txt

```diff
@@ -4,8 +4,8 @@
 Instructions for Installing and running the ISCO_pod_sampler code for the first time
 **********************
 
-1.Download arduino installer-->https://www.arduino.cc/en/Main/Donate
-2.Download Teensyduino that matches your version of Arduino (most likely 1.8.x where x is a number) 
+1.Download and install Arduino-->https://www.arduino.cc/en/Main/Donate
+2.Download and install Teensyduino that matches your version of Arduino (most likely 1.8.x where x is a number) 
```

Comment:
@hawes1010 should we make these install instructions version-agnostic?

Reply:
I like the change